README.md

```diff
@@ -28,6 +28,20 @@ template_foo: "oof"
 template_bar: "baz"
 ```
 
+## Variables Exported by the Role
```

Comment:
Just noticed that the previous section on the same level is also about variables. Should this maybe be a sub section (`###` instead of `##`).